locales/wxMaxima/fr.po

```diff
@@ -2041,7 +2041,7 @@ msgstr "Entrer une matrice"
 #: ../../src/ConfigDialogue.cpp:712
 #, fuzzy
 msgid "Enter adds newlines, Ctrl+Enter evaluates cells"
-msgstr "Évaluation des cellules avec la touche Entrée"
+msgstr "Évaluation des cellules avec la touche Ctrl+Entrée"
```

Comment:
Please remove "#, fuzzy" to unfuzzy the translation. A similar operation should be used for other fuzzy strings.